src/NuGet.Core/NuGet.Common/PathUtil/PathUtility.cs

```diff
@@ -467,5 +467,16 @@ private static bool CheckCaseSenstivityRecursivelyTillDirectoryExists(string pat
             }
             return false;
         }
+
+        public static string StripLeadingDirectorySeparators(string filename)
+        {
+            var currentDirectoryPath = $".{Path.DirectorySeparatorChar}";
```

Comment:
what if currentDirectoryPath starts with ./ on windows? this wouldn't strip it in that case even though its a valid path.

Reply:
although i am not convinced of the utility of this method. why is this needed?

Reply:
PackageLicenseFile specifies the file relative to the root of the nupkg. 

If you want to pack a file without an extension, you'd need to do the following: 
```
<PropertyGroup>
    <PackageLicenseFile>LICENSE</PackageLicenseFile>
</PropertyGroup>
 <ItemGroup>
    <None Include="licenses\LICENSE" Pack="true" PackagePath="."/>
</ItemGroup>
```

This strips out the . from the path in the comparison. 

Any other ideas?